client/js/jsonedit.js

```diff
@@ -2088,8 +2089,9 @@ window.addEventListener('mousemove', function(e) {
     widgetCoordCache = [];
     for(const widget of widgets.values()) {
       const coords = widget.coordGlobalFromCoordParent({x:widget.get('x'),y:widget.get('y')});
-      coords.r = coords.x + widget.get('width');
-      coords.b = coords.y + widget.get('height');
+      const w = '#w_'+widget.id;
+      coords.r = coords.x + $(w).offsetWidth;
+      coords.b = coords.y + $(w).offsetHeight;
```

Comment:
Apart from the fact that I still don't think it's a good idea to ignore `width` and `height`, this code change causes crashes in the JSON editor (#1385) because it doesn't escape the `id`. Should be `const w = '#w_'+escapeID(widget.id);` or something like that.

You can just access `widget.domElement` though, now that I think about it for a second.